loomi/regulariza-morador-client
# Refactor/delete cards home

## Feature
 - Delete cards from landing page
 - fix responsivity

**How to test:**

1. In your browser go to [stage](http://project.netlify.app/) or [localhost](http://localhost:3000/)
2. Check if the component/view looks just like the [mockup](https://xd.adobe.com/) in different screen sizes (width between 1024px and 1920px)
3. Test the things that you think are worth testing, even the ones that are not in this description

![image](https://user-images.githubusercontent.com/32523401/135313213-87d7cbf1-c8cf-4c5e-8351-b59b44f4bf48.png)
